week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -6,9 +6,15 @@
   They have also decided that they should add a 1% fee to all foreign transactions
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
-
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function transactionFee(price) {
+  return price * 0.01;
+}
+function convertToUSD(price) {
+  return price * 1.4 + transactionFee(price);
+}
+function convertToBRL(price) {
+  return price * 4.85 + transactionFee(price);
+}
 
```

Comment:
See previous comments. Going forward, do you think that it would be considered good practice to hard code specific numbers into your functions like this? What would be the consequence of the exchange rateor the transaction fee, changing for example?